### PR TITLE
Behzad's week2 exercises

### PR DESCRIPTION
**Pull request by Behzadkk**
I've done the first week of JS exercises.
Newcastle branch.
Thanks

### CODE REVIEW

File: week-1-practise/level-1/10-function-output-3.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 function concatenate(firstWord, secondWord, thirdWord) {
-    // Write the body of this function to concatenate three words together
+  // Write the body of this function to concatenate three words together
+  return firstWord.concat(secondWord, thirdWord);
```
Comment (amaalali) [inline week-1-practise/level-1/10-function-output-3.js]: I like that you noticed that `concat` takes multiple params!

File: week-1-practise/level-1/5-logic-error-2.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return word.trim;
+  return word.trim();
```
Comment (amaalali) [inline week-1-practise/level-1/5-logic-error-2.js]: :+1: Did you notice what `word.trim` returned?

File: week-1-practise/level-1/8-function-output-1.js [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
-// - What does this program do?
-// - How many parameters does the function take?
-// - What is the function name?
-// - Where is the function called? with what parameters?
\ No newline at end of file
+// - What does this program do? It finds a random number between 0 and 1, then multiply in 10
```
Comment (amaalali) [inline week-1-practise/level-1/8-function-output-1.js]: 👍

File: week-1-practise/level-3/1-piping.js [hunk 4]
```diff
@@ -11,7 +11,10 @@
 */
 
 var startingValue = 5;
-
+function piping(value) {
```
Comment (amaalali) [inline week-1-practise/level-3/1-piping.js]: Not quite, we want to use the `add`, `multiply` and `format `  functions below

Comment (Behzadkk) [reply week-1-practise/level-3/1-piping.js]: Yes. I noticed when I had sent the pull request. I have fixed my codes.

let result = format(add(multiply(startingValue, 2), 10));

**Comment by Behzadkk** [comment]
Hi 
Please find my exercises at the week-2 and week-2- practice folders.
Thanks

**Pull request closed without merging**